app/models/manageiq/providers/ansible_tower/configuration_manager/ansible_tower_job.rb

```diff
@@ -0,0 +1,60 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::AnsibleTowerJob < ::OrchestrationStack
```

Comment:
@bzwei can the AnsibleTowerJob be changed to just Job since we already have AnsibleTower after Providers in the name it will be consistent with others like ConfiguredSystem